### ISSUE TITLE
admin and cluster API

### ISSUE THREAD

**Issue by rlouapre** [ryanjdew/go-marklogic-go#2]
Are admin and cluster APIs implemented?

If not I am happy to work on them.

**Comment by ryanjdew** [comment]
No, the admin and cluster APIs aren't implemented. So far only the server properties and database properties from the management API have been implemented.
